src/widgets/bar/lv_bar.h

```diff
@@ -109,6 +120,13 @@ void lv_bar_set_range(lv_obj_t * obj, int32_t min, int32_t max);
  */
 void lv_bar_set_mode(lv_obj_t * obj, lv_bar_mode_t mode);
 
+/**
+ * Set the orientation of bar.
+ * @param obj       pointer to bar object
+ * @param orient    bar orientation from ::lv_bar_orientation_t
```

Comment:
You could write full parameter name: `orientation`

Reply:
I agree!